src/Esri.ArcGISRuntime.Toolkit/WPF/Esri.ArcGISRuntime.Toolkit.WPF.csproj

```diff
@@ -22,6 +22,7 @@
       <IncludeAssets>compile</IncludeAssets>
       <PrivateAssets></PrivateAssets>
     </PackageReference>
+    <PackageReference Include="System.ValueTuple" Version="4.4.0" />
```

Comment:
Is this necessary? It seems like a lot of a single line where it's used.
Also I believe the nuspec should declare the dependency too